athena-neptune/docs/aws-glue-sample-scripts/RDF.md

```diff
@@ -22,30 +22,6 @@ In each case, you define columns and use table properties to map RDF to that col
 ## Examples
 We provide examples of both class-based and query-based tables. The examples use the Air Routes dataset. 
 
-### Step 1: Create Neptune Cluster and Seed Air Routes Data in Neptune
```

Comment:
why are we removing the documentation for the seeding process?

Reply:
The documentation has been restructured. but all important detail is still there in the docs. We did decide, to make the example easier to follow, to use CloudFormation rather than CLI and SDK to deploy the Glue catalog.